platform/ios/Integration Tests/MGLMapViewIntegrationTest.m

```diff
@@ -53,7 +50,17 @@ - (MGLMapView *)mapViewForTestWithFrame:(CGRect)rect styleURL:(NSURL *)styleURL
 - (void)setUp {
     [super setUp];
 
-    [MGLAccountManager setAccessToken:self.accessToken ?: @"pk.feedcafedeadbeefbadebede"];
+    NSString *accessToken;
+    
+    if ([self.name containsString:@"🔒"]) {
```

Comment:
I was thinking if it makes sense use `hasSufix` instead?

Reply:
We have a case where there are two of these things, so unfortunately it needed to be "contains" (it used to be `hasSuffix`).